Source/Scene/Scene.js

```diff
@@ -945,6 +950,36 @@ define([
         },
 
         /**
+         * This property is for debugging only; it is not for production use.
+         * <p>
+         * When <code>true</code>, draws outlines to show the boundaries of the camera frustums
+         * </p>
+         *
+         * @type Boolean
+         *
+         * @default false
+         */
+        debugShowFrustumPlanes : {
```

Comment:
When this is merged into the `3d-tiles` branch, please remove the code that explicitly creates the debug frustum in the `Cesium3DTileset` and just have the inspector set this property.